t/46-containters.t

```diff
@@ -0,0 +1,6 @@
+use Test::Most;
+use Test::Warnings ':report_warnings';
+
+is(system('docker', 'build', '-t', 'openqa_worker', 'docker/worker'), 0);
+is(system('docker run -t -v $(pwd)/data/workers.ini:/etc/openqa/workers.ini openqa_worker | grep "API key and secret are needed" | wc -l'), 0, '1');
```

Comment:
As @Martchus mentioned tests should have a description. See https://perldoc.perl.org/Test::More for details. All `Test::More` test commands have a similar syntax, e.g.

```perl
is $got, $expected, $test_status_message_when_this_step_is_called;
```

For example you could write:

```perl
is system('docker build -t openqa_worker docker/worker'), 0, 'worker container image can be built';
…
```

Reply:
done